src/staff-administrator/components/staff-user-details/staff-user-details.component.spec.ts

```diff
@@ -220,10 +221,11 @@ describe('StaffUserDetailsComponent', () => {
   it('should set suspendedStatus to "suspended" to show the banner when calling updateUserStatus with isSuspended true', () => {
     expect(component.userDetails.suspended).toBe(false);
     mockStaffDataAccessService.updateUser.and.returnValue(of({ case_worker_id: '123' }));
+    mockStaffDataAccessService.fetchSingleUserById.and.returnValue(of(StaffUser.from(testStaffUserData)));
     component.updateUserStatus();
 
     expect(mockStaffDataAccessService.updateUser).toHaveBeenCalled();
-    expect(component.userDetails.suspended).toBe(true);
+    expect(component.userDetails.suspended).toBe(false);
```

Comment:
So I understand the test is a little confusing, what happens when the user is updated is that we get the user details again afterwards instead of setting the suspended field ourselves. The mock was using data with an active status, hence you would not see suspended in the test. I've changed this to resolve this and make the test more legible

Reply:
OK, that makes more sense - thanks, @connorpgpmcelroy